Formula/dvc.rb

```diff
@@ -699,8 +699,8 @@ def install
 
     virtualenv_install_with_resources
 
-    (bash_completion/"dvc").write Utils.safe_popen_read(bin/"dvc", "completion", "-s", "bash")
-    (zsh_completion/"_dvc").write Utils.safe_popen_read(bin/"dvc", "completion", "-s", "zsh")
+    generate_completions_from_executable(bin/"dvc", "completion",
+                                         shells: [:bash, :zsh], shell_parameter_format: "-s=")
```

Comment:
What about just doing this for consistency? (this works, right?)

```suggestion
    generate_completions_from_executable(bin/"dvc", "completion", "-s", shells: [:bash, :zsh])
```

Reply:
Yes, this works. Looks better, my initial commit must have been before we made commands into a vararg.